Makefile

```diff
@@ -31,26 +32,19 @@ endif
 
 ## -- Docker image
 
-# for dev, on GKE, use GCR and GCLOUD_PROJECT
-ifneq ($(findstring gke_,$(KUBECTL_CLUSTER)),)
-	REGISTRY ?= eu.gcr.io
-else
-	# default to local registry
-	REGISTRY ?= localhost:5000
-endif
-
 # for dev, suffix image name with current user name
 IMG_SUFFIX ?= -$(subst _,,$(shell whoami))
 
-REPOSITORY  ?= eck
+REGISTRY    ?= docker.elastic.co
+NAMESPACE   ?= eck-dev
```

Comment:
Should we name it `REGISTRY_NAMESPACE` (and `E2E_REGISTRY_NAMESPACE`)  to avoid confusion with k8s namespaces?

Reply:
Good idea.